gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java

```diff
@@ -42,6 +42,10 @@
 @InternalApi
 @AutoValue
 public abstract class EndpointContext {
+  public static final String INVALID_UNIVERSE_DOMAIN_ERROR_TEMPLATE =
```

Comment:
Can these two templates be private or package private?

Reply:
I made `INVALID_UNIVERSE_DOMAIN_ERROR_TEMPLATE` private, but `UNABLE_TO_RETRIEVE_CREDENTIALS_ERROR_MESSAGE` needs to be public as it's accessed from the gRPC and HttpJson sub-modules.